src/02-crm/companies.apib

```diff
@@ -199,3 +199,15 @@ Delete a company.
         + id: `f3d67f3e-b8a9-45e8-99b1-17a3b14de8a6` (string, required)
 
 + Response 204
+
+### companies.tag [POST /companies.tag]
+
+Add a new or existing tag to a company.
```

Comment:
Can you double check with the implementation if this is correct? From what I remember we would release without the ability to create new tags first?

Reply:
Already implemented it **with** adding new tags, so I guess this can be merged as is (when the implementation is released)